### PR TITLE
Django feed urls

### PR DESCRIPTION
**Pull request by sagelliv**
Added feed urls: a way for users to specify a url (drive, gist, dropbox, etc.) with newline separated urls for Portia to crawl.

### CODE REVIEW

File: slybot/slybot/starturls/feed_generator.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from scrapy import Request
```
Comment (ruairif) [inline slybot/slybot/starturls/feed_generator.py]: Can the slybot changes be moved to a different PR so they can be released on PyPi

File: slybot/slybot/starturls/__init__.py [hunk 2]
```diff
@@ -6,16 +6,16 @@
 import six
 from six.moves.urllib_parse import urlparse
 
+from .feed_generator import FeedGenerator
```
Comment (ruairif) [inline slybot/slybot/starturls/__init__.py]: Is there a reason to import FeedGenerator?

Comment (sagelliv) [inline slybot/slybot/starturls/__init__.py]: So that we can have `from slybot.starturls import FeedGenerator`. The `__init__.py` file is used here to expose what we want from the python package.

**Comment by sagelliv** [comment]
Depends on https://github.com/scrapinghub/portia/pull/587 and needs a rebase with `nui-develop` to work.

File: slybot/slybot/starturls/feed_generator.py (hunk 1 above)
Comment (sagelliv) [inline slybot/slybot/starturls/feed_generator.py]: Done

File: portiaui/app/models/start-url.js [hunk 3]
```diff
@@ -33,41 +67,48 @@ export default function startUrl(spec) {
...
-            base.fragments = urlObject.fragments;
+        if (url.includes('dropbox.com') && notRaw) {
+            return url + '&raw=1';
         }
-        return base;
-    }
 
-    let urlObject = buildUrlObject();
-    urlObject.isGenerated = urlObject.type === 'generated';
+        if (url.includes('google.com') && !url.includes('export')) {
```
Comment (sagelliv) [inline portiaui/app/models/start-url.js]: Bitbucket snippets are tricky to get right. I opted for going with the url `http://files.scrapinghub.com/portia/urls.txt`.
